ToDoApi/Controllers/ToDoController.cs

```diff
@@ -15,12 +16,39 @@ public IActionResult CreateToDo([FromBody] TodoItemRequest request)
             {
                 request.Validate();
 
-                return Ok(new TodoItem(request.Title, request.Description, request.EndDate));
+                var todo = new TodoItem(request.Title, request.Description, request.EndDate);
+                TodoList.todoItems.Add(todo);
+
+                return Ok(todo);
             }
             catch(BadHttpRequestException ex)
             {
                 return BadRequest(new { message = ex.Message });
             }
         }
+
+        [HttpGet]
+        public ActionResult<List<TodoItem>> GetAllTodo()
+        {
+            if (TodoList.todoItems.Count == 0)
```

Comment:
```suggestion
            if (TodoList.todoItems.Count == 0)
```
Essas validações de lista sempre são bem feias por conta própria, sugiro a criação de um método na classe TodoList, que faça essa validação,  dessa forma isso ficaria **mais ou menos assim**:

```c#
            if (TodoList.IsEmptyTodoItems())
```